doc/source/getting_started/install.rst

```diff
@@ -335,7 +335,8 @@ lxml                      4.9.2              xml             XML parser for read
 SQL databases
 ^^^^^^^^^^^^^
 
-Installable with ``pip install "pandas[postgresql, mysql, sql-other]"``.
+Traditional drivers are installable with ``pip install "pandas[postgresql, mysql, sql-other]"``.  ADBC drivers
```

Comment:
I think this is no longer anymore since you added them to the `postgresql` and `sql-other` extra in the pyproject.toml